tests/cases/fourslash/completionsInJsxTag.ts

```diff
@@ -11,17 +11,17 @@
 ////            foo: string
 ////        }
 ////    }
-////}
-////class Foo {
-////    render() {
-////        <div /*1*/ ></div>;
-////        <div  /*2*/ />
-////    }
+////}
+////class Foo {
+////    render() {
+////        <div /*1*/ ></div>;
+////        <div  /*2*/ />
+////    }
```

Comment:
I think this change was due to mixed line endings in the file.  I can remove this part of the change if needed.